### PR TITLE
[WIP] Adds gender fluid

### PR DESCRIPTION
**Pull request by deathride58**
Title. In celebration of pride month, this PR adds gender fluid to the game! Gender fluid is a reagent made with 3 plasma, 1 phenol, and 1 lube. Every process cycle that it's in your system, it randomly switches your gender between male, female, and plural.

## Changelog
:cl:
add: Happy pride month! Added gender fluid to the game. It's made with 3 plasma, 1 lube, and 1 phenol.
/:cl:

todo:

- [ ] Make it reset the user's gender back to what it's set to in their DNA once it metabolizes out.

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Why make it about pride month?

**Comment by MrJWhit** [comment]
So why is lube included in it?

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 1]
```diff
@@ -441,3 +441,15 @@
...
 	..()
 
+/datum/reagent/drug/genderfluid
+	name = "Gender Fluid"
+	id = "genderfluid"
+	description = "A curious fluid that causes a patient's gender to become wildly inconsistent!"
+	reagent_state = LIQUID
+	color = "#9F60EB"
+	taste_description = "gender politics"
+	addiction_threshold = 10
```
Comment (Ghommie) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:451]: this currently does nothing beside giving the user a negative moodlet and craving messages. Intended?

**Review by Ghommie — COMMENTED** [review]
This just reminds me how gender change xenobio potion is outdated as heck and colorful reagent is hella slower to make.

**Comment by EldritchSigma** [comment]
Thats exactly the kind of gender fluid I expected from the title.. hah.

**Comment by deathride58** [comment]
@Trilbyspaceclone its a meme

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm (hunk 1 above)
Comment (deathride58) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm:451]: Non-harmful addiction is intended, yes. It's there primarily for flavor, as self-discovery is one hell of an addicting drug.

Comment (Ghommie) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm:451]: >
> self-discovery is one hell of an addicting drug.

Well, I'd like to differ but ok, go ahead.

**Comment by Ghommie** [comment]
Pride Month is over, this is stale now.

**Comment by deathride58** [comment]
FUCK

**Comment by deathride58** [comment]
forgot to close this, shit

**Pull request closed without merging**